HelloTest.java

```diff
@@ -0,0 +1,27 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+
+public class HelloTest {
+	private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+	private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+	@Before	
+	public void setUpStreams() {
+   		System.setOut(new PrintStream(outContent));
+		System.setErr(new PrintStream(errContent));	
+	}
+	@After
+	public void cleanUpStreams() {
+    		System.setOut(null);
```

Comment:
This "breaks" System.out for anybody else. Each test should build up its own internal data and then restore things to as they were before the test.